components/sections/repo-questions.js

```diff
@@ -87,10 +113,56 @@ const RepoQuestions = ({ data }) => {
       {data.repo_question.map((q, i) => {
         if (i + 1 === questionToShow) {
           return (
-            <div className="repo-questions" key={q.question}>
-              <Markdown className="repo-questions">{q.question}</Markdown>
-              <br></br>
-              {showOptions && (
+            <div key={q.question}>
+              <div>
+                {questionToShow}. <Markdown inline>{q.question}</Markdown>
+              </div>
+              <br />
+
+              {q.options.length > 2 && (
+                <div>
+                  <FormGroup>
+                    {q.options.map((o, index) => (
+                      <FormControlLabel
+                        key={`checkbox-${index}`}
+                        control={
+                          <Checkbox
+                            onChange={(e) =>
+                              handleCheckboxChange(o, e.target.checked)
+                            }
+                            checked={!!selectedCheckboxes[o.yes_no]}
+                          />
+                        }
+                        label={o.yes_no}
+                        sx={{
+                          "& .MuiFormControlLabel-label": {
+                            marginBottom: 0,
+                          },
+                        }}
+                      />
+                    ))}
+                  </FormGroup>
+
+                  <Button
+                    variant="contained"
+                    onClick={() => setShowRepositories(true)}
+                    disabled={Object.keys(selectedCheckboxes).length === 0}
+                    style={{ marginTop: "10px", color: "white" }}
+                  >
+                    See Repositories
```

Comment:
This button seems like it should be a toggle for a dropdown (or maybe an actual dropdown / accordion component should be used). You click it once and it presents the information, but if you click it again it does nothing. At the very least, I think the button should be hidden or disabled after the first click.